tests/test_wsgiref_inputwrapper_with_size.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+import falcon
+from falcon import testing
+
+
+class TypeResource(testing.SimpleTestResource):
+    """ A simple resource to return the posted request body """
```

Comment:
Style/nitpick: Remove leading and trailing whitespace of the tripple-quoted string, and end the first line with a period.